frontend/src/components/report/CSVReportFormatter.ts

```diff
@@ -102,15 +103,19 @@ export class CSVReportFormatter{
     console.log('RESULT HEADERS', result)
     return result
   }
-  private append_metadata_values(index: number){
-    if(!this.values_metadata){
+
+  private append_values(
+      values: any[],
+      index: number){
+
+    if(!values || values.length == 0){
       return null
     }
     let result = ''
-    let row_keys = Object.keys(this.values_metadata[index])
+    let row_keys = Object.keys(values[index])
```

Comment:
I think here we should use **const** since we do not change a value afterwards

Reply:
Yes